src/utils/ticket/GitHubTicketStatusFetcher.php

```diff
@@ -23,11 +23,6 @@ final class GitHubTicketStatusFetcher implements TicketStatusFetcher
     private string $defaultRepo;
     private ?string $accessToken;
 
-    /**
-     * @var array<string, ?string>
-     */
-    private array $cache;
```

Comment:
we no longer need the caching, because the fetcher now gets a full list of all ticket-keys without duplicates